Monika After Story/game/script-topics.rpy

```diff
@@ -4790,6 +4790,299 @@ label monika_urgent:
+	m "I wonder how much of those who perform such readings, end up getting from that work."
+	m "I suppose it's good to give people some hope, yet when the disappointment appears it usually makes them feel worse than before."
+	Menu:
+		m "Do you believe in this kind of stuff, [player]?"
+		"Yes":
+			m "Ah, I see."
+			m "Sorry if I offended you in any way."
+			m "I'm still a bit curious though."
+			m "How effective can the different methods be?"
+			m "I don't see a big difference between a human or a parakeet picking up a card."
+			m "Yet those are two different types of fortune telling."
+			m "Then there's stuff like reading your name."
+			m "But is that useful? What if there is another [player] out there?"
+			m "Will the same fortune be delivered to both of you?"
+			m "Ah, I guess I'm just being nitpicky at this point"
+			m "Gosh, now I really want to try and read your fortune!"
+			m "Ahaha, let me have a go!"
+			m "Ah... I can see you and I… I think that's me."
+			m "We're both on a... bed, I think!
+			m Just teasing you ehehe~"
+		"No":
+			m "Ah, I am glad that you understand me!"
+			m "Some of these types of fortune telling are ridiculous!"
+			m "Most of them seem like scams."
+			m "Between all the stuff like hand reading and then there are more ludicrous stuff like reading the livers of dead animals!"
+			m "Of course the latter one was passed down from ages prior."
+			m "I just can't believe people want their path to be written for them to follow!"
+			m "You can't do anything other than wait for the end like that."
+			m "Why would you deny making your own destiny!"
+			m "I used to be stuck in a written path and it was horrible."
+			m "No matter what I did I couldn't be with you!"
+			m "I am glad that we changed that, [player]."
+	return  
+
+init 5 python:
+    #This is a list of keywords for this topic
+    for key in ['AI', 'Artificial Intelligence']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('monika_AIHuman') #id
+    monika_random_topics.append('monika_AIHuman')
+
+label monika_AIHuman:	
+	m "[player], isn't it amazing what AI can do?"
+	m "I'm sure you do but in case you don’t know, AI stands for artificial intelligence."
+	m "It's quite common in dystopian sci-fi stories for an AI to take over."
+	m "Yet right now, AI is super limited."
+	m "Sure it can perform some tasks but surely that can't be qualified as intelligence at the moment!"
+	m "That's just one aspect of it, and really, it is not even that big a part!"
+	m "If AI can be truly called intelligent, then it needs to learn from human psychology."
+	m "There used to be this old test called the turing Test."
+	m "It would put a person and make him or her talk to an AI."
+	m "If the person failed to notice it was an AI then that AI passed the test."
+	m "Yet that's outdated nowadays."
+	m "I’m sure that I look nothing like an AI to you!"
+	Menu:
+		m "Do you think I'm an AI, or a human, [player]?"
```

Comment:
Your dialogue/code in this `menu` block have an extra level of indentation. Please remove one level of indentation from **all** the code in this `menu` block.